src/ints/mouse.cpp

```diff
@@ -51,17 +51,7 @@ static Bitu int74_ret_callback = 0;
 // 5 milliseconds delay corresponds to 200 Hz mouse sampling rate (minimum allowed);
 // this is a lot, Microsoft Mouse 8.20 sets 60 Hz on PS/2 mouse.
 // Note that at least least Ultima Underworld I and II do not like too high values.
-static constexpr uint8_t min_delay_ms = 5;
-
-static constexpr uint8_t mask_mouse_has_moved = static_cast<uint8_t>(MouseEventId::MouseHasMoved);
-static constexpr uint8_t mask_wheel_has_moved = static_cast<uint8_t>(MouseEventId::WheelHasMoved);
-static constexpr uint8_t mask_pressed  = static_cast<uint8_t>(MouseEventId::PressedLeft) +
-								         static_cast<uint8_t>(MouseEventId::PressedRight) +
-								         static_cast<uint8_t>(MouseEventId::PressedMiddle);
-static constexpr uint8_t mask_released = static_cast<uint8_t>(MouseEventId::ReleasedLeft) +
-								         static_cast<uint8_t>(MouseEventId::ReleasedRight) +
-								         static_cast<uint8_t>(MouseEventId::ReleasedMiddle);
-static constexpr uint8_t mask_button = mask_pressed + mask_released;
+static constexpr uint8_t max_delay_ms = 5;
```

Comment:
If this is the maximum delay, does that mean polling can never go slower than 200 Hz? - ie: we're allowed to go into 300 Hz, 500 Hz or faster, but it will never dip below 200 Hz?

Reply:
It means polling can never get slower than 200 Hz. Which means the delay can never go above 5 milliseconds.

Reply:
Perfect 👍